install_config/master_node_configuration.adoc

```diff
@@ -1434,6 +1423,37 @@ start on a machine that does not have docker started.
 
 |===
 
+[[node-config-volume-config]]
+=== Local Storage Configuration
+
+You can use the link:https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/7/html/storage_administration_guide/ch-xfs[XFS quota subsystem] 
+to limit the size of `emptyDir` volumes and volumes based on an `emptyDir` volume, such as secrets and configuration maps, on each node.
+
+To limit the size of `emptyDir` volumes in an XFS filesystem, configure local volume quota for each unique 
+xref:../install_config/persistent_storage/pod_security_context.adoc#fsgroup[FSGroup] using the *_node-config-compute_* configuration map in the *openshift-node* project.
+
+----
+apiVersion: kubelet.config.openshift.io/v1
+kind: VolumeConfig
+  localQuota: <1>
+    perFSGroup: 1Gi <2>
+----
+
+<1> Contains options for controlling local volume quota on the node.
+<2> Set this value to a resource quantity representing the desired quota per 
+[FSGroup], per node, such as 1Gi, 512Mi, and so forth. 
```

Comment:
I'd set the values off in backticks to emphasize that they're example parameter values. (Otherwise, you need spaces between the value and abbreviation because of Style.)